source/blog/2016-03-04-atomic-site-on-atomic-app.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Projectatomic.io Running in Atomic App
+author: jberkus
+date: 2016-03-04 12:00:00 UTC
+tags: atomic.app, Nulecule, testing, development
+published: true
+comments: true
+---
+
+Since Atomic.app is getting close to its 1.0 release, I decided it was past time to make the atomic-site into an atomic.app instead of using a shell script that wraps Docker to test it. The new setup is a big improvement, and a useful guide to "Nuleculeizing" your own apps.
+
+As you know, the purpose of Atomic.app and Nulecule is to give you a provider-agnostic way to specify multi-container applications and orchestration metatdata that stays with the application image(s).  Eventually, this will allow for single-command deploys of even large, scalable apps involving many containers.  For now, it lets us get rid of some hackish shell scripting around Docker in our atomic-site test setup.
```

Comment:
Argh, I'm a bit confused. Are we suppose to have it as "atomic app" or "atomic.app"? :confused: